src/Middleware/Rewrite/src/PublicAPI.Shipped.txt

```diff
@@ -16,12 +16,12 @@ Microsoft.AspNetCore.Rewrite.RuleResult.ContinueRules = 0 -> Microsoft.AspNetCor
 Microsoft.AspNetCore.Rewrite.RuleResult.EndResponse = 1 -> Microsoft.AspNetCore.Rewrite.RuleResult
 Microsoft.AspNetCore.Rewrite.RuleResult.SkipRemainingRules = 2 -> Microsoft.AspNetCore.Rewrite.RuleResult
 ~Microsoft.AspNetCore.Rewrite.IRule.ApplyRule(Microsoft.AspNetCore.Rewrite.RewriteContext context) -> void
-~Microsoft.AspNetCore.Rewrite.RewriteContext.HttpContext.get -> Microsoft.AspNetCore.Http.HttpContext
-~Microsoft.AspNetCore.Rewrite.RewriteContext.HttpContext.set -> void
-~Microsoft.AspNetCore.Rewrite.RewriteContext.Logger.get -> Microsoft.Extensions.Logging.ILogger
-~Microsoft.AspNetCore.Rewrite.RewriteContext.Logger.set -> void
-~Microsoft.AspNetCore.Rewrite.RewriteContext.StaticFileProvider.get -> Microsoft.Extensions.FileProviders.IFileProvider
-~Microsoft.AspNetCore.Rewrite.RewriteContext.StaticFileProvider.set -> void
+Microsoft.AspNetCore.Rewrite.RewriteContext.HttpContext.get -> Microsoft.AspNetCore.Http.HttpContext!
```

Comment:
Is `~` supposed to be removed here?

Reply:
Yes. No `~` means it is annotated.

Reply:
IIRC, `~` means that the signature contains at least one reference type which is not known to be either nullable or non-nullable